### DISCUSSION TITLE
Docker-compose for LocalAI CUDA 12 autoinstall WITHOUT MODELS

### DISCUSSION THREAD

**Discussion by ghost** [mudler/LocalAI#2167] [General]
Hi all,
someone can help mi to build a docker-compose.yaml file for build a LocalAI version that support CUDA 12 WITHOUT pre-installed models? Thanks a lot to all!

**Comment by mudler** [comment]
Pre installed models are only present in the All-in-One images. You can use the standard images, that does not have any pre-installed model, see https://localai.io/docs/reference/container-images/ for a list depending on your HW.

**Reply by ghost** [reply]
Hi, where I can recover configuration files for pre-installed models in AIO images? My intentions is to compile a new version with a subset of these models, renamed and reconfigured.
Wait for a response, thanks a lot

**Reply by mudler** [reply]
The AIO models configurations files are in: https://github.com/mudler/LocalAI/tree/master/aio - you can also just drop the files in your models folder

**Reply by ghost** [reply]
Clear. thanks.
Now I placed my YAML files inside the models folder, and I expected it to download automatically, but that doesn't happen. What am I doing wrong?
Thanks and sorry to bother you, but I'm new to this topic

**Comment by BradKML** [comment]
Seconding this for @mudler but for adding SOTA models like LLaMA3 or maybe CodeQwen + DeepSeekCoder, directly from HuggingFace?
